glue/core/hub.py

```diff
@@ -175,9 +184,13 @@ def _find_handlers(self, message):
             # narrow to the most-specific message
             candidate = max(messages, key=_mro_count)
 
-            handler, test = subscriptions[candidate]
+            handler, test, priority = subscriptions[candidate]
             if test(message):
-                yield subscriber, handler
+                priorities.append(priority)
+                prioritized_handlers.append((subscriber, handler))
```

Comment:
I'd rather keep the priorities in the same container as the handlers, so a better way might be to do something like ``prioritized_handlers.append((subscriber, handler, priority))`` then do a ``sort(prioritized_handlers, key=x[2], reversed=True)`` below